setup

```diff
@@ -0,0 +1,212 @@
+    core_count = mp.cpu_count()
+    subprocess.run(["make", f"-j{core_count}"])
+
+    # Transfer files to the query modules directory
+    so_files = glob.glob("**/*.so", recursive=True)
+    for file in so_files:
+        shutil.copy2(file, QUERY_MODULES_DIRECTORY)
+
+    #################################################
+    #             Load Python modules               #
+    #################################################
+
+    os.chdir(PY_DIRECTORY)
+    ignore_list = [
+        "tests",
+        "requirements.txt",
+        "pytest.ini",
+        "htmlcov",
+        "__pycache__",
+        ".*",
+    ]
+
+    # Copy python repository tree inside dist folder
+    copytree(PY_DIRECTORY, QUERY_MODULES_DIRECTORY, ignore_list)
+    os.environ["PYTHONPATH"] = PY_DIRECTORY
+
+    #################################################
+    #              Load Rust modules                #
+    #################################################
+
+    os.chdir(RS_DIRECTORY)
+    for project in filter(
+            lambda f: os.path.isdir(f) and f != "rsmgp-sys",
+            os.listdir(RS_DIRECTORY)):
+        project_dir = os.path.join(RS_DIRECTORY, project)
+        os.chdir(project_dir)
+        subprocess.run(["cargo", "build", "--release"])
+
+        release_dir = os.path.join(project_dir, "target", "release")
+        os.chdir(release_dir)
+        modules = glob.glob("*.so")
+        assert len(modules) == 1, f"Bad rust/{project} lib."
+
+        module = modules[0]
+        src_file = os.path.join(release_dir, module)
+        module = module.lstrip("lib") if module.startswith("lib") else module
+        dst_file = os.path.join(QUERY_MODULES_DIRECTORY, module)
+        shutil.copy2(src_file, dst_file)
+
+
+def main():
+    arguments = get_arguments()
+
+    if not hasattr(arguments, 'action'):
+        log.log(level=log.INFO, msg=HELP_MESSAGE)
+        return
+
+    if arguments.action == "build":
+        log.log(level=log.INFO, msg="Starting building of source code...")
+        build()
+        log.log(level=log.INFO, msg="Building done.")
+        if arguments.path is not None:
+            log.log(level=log.INFO,
+                    msg="Skipping copy of contents of 'dist' folder to specified folder since --path is not specified")
+            return
+
+        log.log(level=log.INFO,
+                msg='Copying files from folder {src} to {dst} folder.'.format(src=QUERY_MODULES_DIRECTORY,
+                                                                              dst=arguments.path))
+        copy_dist(arguments.path)
+        log.log(level=log.INFO, msg='Copying done!')
+
+        return
+
+    if arguments.action == "modules":
+        if arguments.path is None:
+            path = f"{pathlib.Path().resolve()}/dist"
+        else:
+            path = arguments.path
+
+        if arguments.mg_conf_path:
+            mg_conf_path = arguments.mg_conf_path
+        else:
+            mg_conf_path = "/etc/memgraph/memgraph.conf"
```

Comment:
Logic is different for these two. At first, default item is in the first `if` branch, and later it is under else `branch`